### PR TITLE
hollow knight plush

### PR DESCRIPTION
**Pull request by ynot01**
:cl:  
rscadd: Added a hollow plushie
/:cl:

### CODE REVIEW

File: yogstation/code/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -5,6 +5,12 @@
 	desc = "A small stuffed doll of the elder goddess Nar'Sie. Who thought this was a good children's toy? <b>It looks sad.</b>"
 	is_invoker = FALSE
 
+/obj/item/toy/plush/hollow
+	name = "hollow plushie"
+	icon = 'yogstation/icons/obj/toy.dmi'
+	icon_state = "theknight"
+	desc = "Ident is fucking stupid"
```
Comment (Amelia0010) [inline yogstation/code/game/objects/items/plushes.dm:13]: 🤔

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

File: yogstation/code/game/objects/items/plushes.dm (hunk 1 above)
Comment (ynot01) [reply yogstation/code/game/objects/items/plushes.dm:13]: cwu3KPb

**Comment by nesisknesis** [comment]
wow great pr. as head of hollow knight here at yogstation incorporated i approve of this pr.

edit: if you turn the thumb upright its a thumbs up

**Comment by nesisknesis** [comment]
after a recent commit i recede all support for this pr. i rate this pr to have sucked massive chonga

**Comment by ynot01** [comment]
why don't you want any interaction at all you dong

**Comment by nesisknesis** [comment]
thank you travis at least someone is cultured in quality hollow knight prs

**Comment by ynot01** [comment]
you think i compiled this smh

File: yogstation/code/game/objects/items/plushes.dm [hunk 2]
```diff
@@ -5,6 +5,14 @@
 	desc = "A small stuffed doll of the elder goddess Nar'Sie. Who thought this was a good children's toy? <b>It looks sad.</b>"
 	is_invoker = FALSE
 
+/obj/item/toy/plush/hollow
+	name = "hollow plushie"
+	icon = 'yogstation/icons/obj/toy.dmi'
+	icon_state = "theknight"
+	attack_verb = list("parried", "slashed at")
+	desc = "Ident is fucking stupid"
+	component = /datum/component/squeak/lizardplushie
```
Comment (alexkar598) [inline yogstation/code/game/objects/items/plushes.dm:14]:
```suggestion
	squeak_override = list('sound/weapons/slash.ogg' = 1)
```

**Review by alexkar598 — CHANGES_REQUESTED** [review]

**Comment by alexkar598** [comment]
img

**Pull request closed without merging**